src/alerting/utils/vis.ts

```diff
@@ -60,6 +60,15 @@ export const useCheckYDomain = (
     return extent(extrema)
   }, [dataDomain, thresholds])
 
+  return initialDomain
+}
+
+// Calls the above method, but allows for the setting of state one way
+export const useCheckYDomainWithState = (
```

Comment:
the `useOneWayState` used to be in the above function (`useCheckYDomain`) which caused problems when creating a check - it would cause infinite re-renders when adding a check